components/user-header.module.css

```diff
@@ -3,24 +3,6 @@
     margin-bottom: 0;
 }
 
-.nav {
-    margin: 1rem 0;
-    justify-content: start;
-    font-size: 110%;
-}
-
-.nav :global .nav-link {
-    padding-left: 0;
-}
-
-.nav :global .nav-item:not(:first-child) {
-    margin-left: 1rem;
```

Comment:
I assume we didn't specifically want to have 1rem gaps in profiles, because the CSS defined in nav.module.css uses 0.5rem gaps via the `gap` property